itou/utils/apis/metabase.py

```diff
@@ -280,6 +281,11 @@ def metabase_embedded_url(request=None, dashboard_id=None, params=None, with_tit
         metabase_dashboard = METABASE_DASHBOARDS.get(view_name)
         dashboard_id = metabase_dashboard["dashboard_id"] if metabase_dashboard else None
 
-    payload = {"resource": {"dashboard": dashboard_id}, "params": params, "exp": round(time.time()) + (10 * 60)}
+    token_duration_in_seconds = 30 * 60
+    payload = {
+        "resource": {"dashboard": dashboard_id},
+        "params": params,
+        "exp": round(time.time()) + token_duration_in_seconds,
```

Comment:
C'est un peu plus verbeux mais aussi plus explicite :).
```suggestion
        "exp": round(timezone.now() + datetime.timedelta(minutes=30).timestamp()),
```

Reply:
J'ai essayé de faire fonctionner sans succès ta suggestion.

- `time.time()` donne le nombre de secondes depuis le 1er janvier 1970.

- en suivant ta suggestion, j'ai cherché pour compter le nombre de secondes de la même manière mais au final `int((datetime.datetime.now() - datetime.datetime(1970, 1, 1)).total_seconds())` donne 2*3600 (2h) secondes de plus que `time.time()` surement en raison de fuseau horaire

Par prudence je reste sur le code qui a fait ses preuves, mais je vais rajouter quelques commentaires.

Reply:
Euh ca ne fonctionne pas tel quel, `round` ne fonctionne pas sur du `datetime` et il manque la logique du fait de compter le nombre de secondes depuis 1970. J'ai réussi à faire fonctionner le tout, c'est nettement plus verbeux mais au moins c'est explicite.

Reply:
Oui, il manquais une paire de parenthèses : `round((timezone.now() + datetime.timedelta(minutes=30)).timestamp())`.
```
In [28]: round(time.time() + 30*60) == round((timezone.now() + datetime.timedelta(minutes=30)).timestamp())
Out[28]: True
```